narwhals/expression.py

```diff
@@ -721,6 +721,55 @@ def fill_null(self, value: Any) -> Expr:
 
     # --- partial reduction ---
     def drop_nulls(self) -> Expr:
+        """
+        Remove missing values.
+
+        Notes:
+            pandas and Polars handle null values differently. Polars distinguishes
+            between NaN and Null, whereas pandas doesn't.
+
+        Examples:
+            >>> import narwhals as nw
+            >>> import pandas as pd
+            >>> import polars as pl
+
+            >>> data = {
+            ...             "a": [2, 4, None, 3, 5],
+            ...             "b": [2.0, 4.0, float("nan"), 3.0, 5.0]
+            ...         }
```

Comment:
instead of making two columns and not using one of them, how about making column `'b'` have both` float('nan')` and `None`?

Reply:
Alright